t/091-coroutine.t

```diff
@@ -483,8 +483,8 @@ done
 GET /lua
 --- response_body
 hello
---- error_log eval
-["stack traceback:", "coroutine 0:", "coroutine 1:", "coroutine 2:"]
+--- no_error_log
+[error]
```

Comment:
`coroutine.resume` is a `pcall` and its purpose is to catch the errors aka silence the errors and let users handle them. I will make that test better so it checks the return value.